src/storage/new_txn/new_txn_index_impl.cpp

```diff
@@ -1371,7 +1376,7 @@ Status NewTxn::PopulateEmvbIndexInner(SharedPtr<IndexBase> index_base,
     ChunkID chunk_id = 0;
     {
         Status status = segment_index_meta.GetNextChunkID(chunk_id);
-        if (!status.ok()) {
+        if (!status.ok() && status.code_ != ErrorCode::kNotFound) {
```

Comment:
There are about 5 places calling `GetNextChunkID` and `SetNextCheckID`. Maybe it's better to instroduce a function to do both, for example `SizeT IncrNextChunkID()`.